compare.repertoires.py

```diff
@@ -176,27 +177,27 @@ def file2dict(fileName):
 
 
 #VJ -----------------------------
+# also remove "+ sample_1" or "+ sample_2" from the following lines
+file_IGH_1=args.dir_1+"/IGH.VJ.FREQ."+".csv"
+file_IGH_2=args.dir_2+"/IGH.VJ.FREQ."+".csv"
 
-file_IGH_1=args.dir_1+"/IGH.VJ.FREQ."+sample_1+".csv"
-file_IGH_2=args.dir_2+"/IGH.VJ.FREQ."+sample_2+".csv"
-
-file_IGK_1=args.dir_1+"/IGK.VJ.FREQ."+sample_1+".csv"
-file_IGK_2=args.dir_2+"/IGK.VJ.FREQ."+sample_2+".csv"
+file_IGK_1=args.dir_1+"/IGK.VJ.FREQ."+".csv"
+file_IGK_2=args.dir_2+"/IGK.VJ.FREQ."+".csv"
 
-file_IGL_1=args.dir_1+"/IGL.VJ.FREQ."+sample_1+".csv"
-file_IGL_2=args.dir_2+"/IGL.VJ.FREQ."+sample_2+".csv"
+file_IGL_1=args.dir_1+"/IGL.VJ.FREQ."+".csv"
+file_IGL_2=args.dir_2+"/IGL.VJ.FREQ."+".csv"
 
-file_TCRA_1=args.dir_1+"/TCRA.VJ.FREQ."+sample_1+".csv"
-file_TCRA_2=args.dir_2+"/TCRA.VJ.FREQ."+sample_2+".csv"
+file_TCRA_1=args.dir_1+"/TCRA.VJ.FREQ."+".csv"
+file_TCRA_2=args.dir_2+"/TCRA.VJ.FREQ."+".csv"
 
-file_TCRB_1=args.dir_1+"/TCRB.VJ.FREQ."+sample_1+".csv"
+file_TCRB_1=args.dir_1+"/TCRB.VJ.FREQ."+".csv"
 file_TCRB_2=args.dir_2+"/TCRB.VJ.FREQ."+sample_2+".csv"
```

Comment:
Is there a reason you didn't take out the sample_2 in this line?

Reply:
oh,my god. I forgot to remove these ones. all sample_1 or sample_2 should be removed

Reply:
Great!

Reply:
Actually, @mandricigor, don't you think it would be better to instead modify the clonality.py file so that the outputs contain the sample names?

Reply:
I merged these changes with the Mangul USC lab fork.
@mandricigor, I recommend merging this patch into your fork.
@dormeight made a good fix.